src/components/application_manager/test/commands/command_impl_test.cc

```diff
@@ -154,59 +160,55 @@ TEST_F(CommandImplTest, ReplaceMobileByHMIAppId_NoAppIdInMessage_UNSUCCESS) {
 
 TEST_F(CommandImplTest, ReplaceMobileByHMIAppId_SUCCESS) {
   MessageSharedPtr msg = CreateMessage();
-  (*msg)[strings::app_id] = kAppId1_;
+  (*msg)[strings::app_id] = kAppId1;
 
   UCommandImplPtr command = CreateCommand<UCommandImpl>(msg);
 
   MockAppPtr app = CreateMockApp();
 
-  EXPECT_CALL(app_mngr_, application(kAppId1_)).WillOnce(Return(app));
-  ON_CALL(*app, hmi_app_id()).WillByDefault(Return(kAppId2_));
+  EXPECT_CALL(app_mngr_, application(kAppId1)).WillOnce(Return(app));
+  ON_CALL(*app, hmi_app_id()).WillByDefault(Return(kAppId2));
 
   command->ReplaceMobileByHMIAppId(*msg);
 
-  EXPECT_EQ(kAppId2_, (*msg)[strings::app_id].asUInt());
+  EXPECT_EQ(kAppId2, (*msg)[strings::app_id].asUInt());
 }
 
 TEST_F(CommandImplTest, ReplaceMobileByHMIAppId_Array_SUCCESS) {
-  MessageSharedPtr msg = CreateArrayMessage(kDefaultMsgCount_);
+  MessageSharedPtr msg = CreateArrayMessage(kDefaultMsgCount);
   UCommandImplPtr command = CreateCommand<UCommandImpl>(msg);
 
   MockAppPtr app = CreateMockApp();
 
   EXPECT_CALL(app_mngr_, application(_))
-      .Times(kDefaultMsgCount_)
+      .Times(kDefaultMsgCount)
       .WillRepeatedly(Return(app));
-  ON_CALL(*app, hmi_app_id()).WillByDefault(Return(kAppId2_));
+  ON_CALL(*app, hmi_app_id()).WillByDefault(Return(kAppId2));
 
   command->ReplaceMobileByHMIAppId(*msg);
 
   EXPECT_TRUE(msg->asArray());
-  smart_objects::SmartArray::const_iterator it(msg->asArray()->begin());
-  const smart_objects::SmartArray::const_iterator it_end(msg->asArray()->end());
-  for (; it != it_end; ++it) {
-    EXPECT_EQ(kAppId2_, (*it)[strings::app_id].asUInt());
-  }
+  std::for_each(
+      msg->asArray()->begin(), msg->asArray()->end(), ExpectEqualAppId);
 }
 
 TEST_F(CommandImplTest, ReplaceMobileByHMIAppId_Map_SUCCESS) {
-  MessageSharedPtr msg = CreateMapMessage(kDefaultMsgCount_);
+  MessageSharedPtr msg = CreateMapMessage(kDefaultMsgCount);
   UCommandImplPtr command = CreateCommand<UCommandImpl>(msg);
 
   MockAppPtr app = CreateMockApp();
 
   EXPECT_CALL(app_mngr_, application(_))
-      .Times(kDefaultMsgCount_)
+      .Times(kDefaultMsgCount)
       .WillRepeatedly(Return(app));
-  ON_CALL(*app, hmi_app_id()).WillByDefault(Return(kAppId2_));
+  ON_CALL(*app, hmi_app_id()).WillByDefault(Return(kAppId2));
 
   command->ReplaceMobileByHMIAppId(*msg);
 
   std::set<std::string> keys(msg->enumerate());
-  std::set<std::string>::const_iterator key(keys.begin());
-  for (; key != keys.end(); ++key) {
-    EXPECT_EQ(kAppId2_, (*msg)[*key][strings::app_id].asUInt());
-  }
+  std::for_each(keys.begin(),
+                keys.end(),
+                std::bind(ExpectEqualKeyAppId, std::placeholders::_1, msg));
```

Comment:
@okozlovlux `bind` - is c++11. We don't use it in our code. Please fix it in all places.